app/models/lti_configuration/moodle_settings.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class LtiConfiguration
+  class MoodleSettings < GenericSettings
+    def initialize
```

Comment:
We shouldn't need this initializer anymore!

Reply:
oop! good catch